sale_automatic_workflow/__manifest__.py

```diff
@@ -5,20 +5,20 @@
 
 {
     "name": "Sale Automatic Workflow",
-    "version": "14.0.1.1.4",
+    "version": "15.0.1.0.0",
     "category": "Sales Management",
     "license": "AGPL-3",
     "author": "Akretion, "
     "Camptocamp, "
     "Sodexis, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
-    "depends": ["sale_stock", "sales_team"],
+    "depends": ["sale_stock", "sales_team", "sale_delivery_state"],
```

Comment:
Isn't it a little bit too much to pull this just for tests ?

Reply:
Is not for tests, is the roadmap. Please check the diff on sale_order.py :wink: 
See discussion on https://github.com/OCA/sale-workflow/pull/1372
We don't need sale_automatic_workflow_delivery_state anymore.